packages/cyberstorm/src/components/componentStyles/Breadcrumb.module.css

```diff
@@ -0,0 +1,24 @@
+/* Common Breadcrumb styles */
+.breadcrumb {
+  display: flex;
+  flex-direction: row;
+  gap: var(--size-s);
+  align-items: center;
+  font-size: var(--size-m);
+  font-family: Inter, sans-serif;
+}
+
+.breadcrumb > a {
+  display: flex;
+  flex-direction: row;
+  gap: var(--size-s);
+  align-items: center;
+  color: var(--color-cyber-green--50);
+  font-weight: 400;
+
+  text-decoration: none;
```

Comment:
Should we add underline when the link is hovered?

Reply:
Wasn't in the designs IIRC. @gkurck What do you think?

Reply:
The designs do have underline on hover though.

Tbf might be good idea to map out the default visual behaviour of anchors with @VilppeRiskidev to see if a default styles for anchors could be added which components like these inherit.

Reply:
Yeah, alright I'm not going to add the underline hover here. I'll communicate with Vili about this 👍